templates/base.html

```diff
@@ -0,0 +1,29 @@
+{% load static %}
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    {% block title %}
+      <title>Taxi Service</title>
+    {% endblock %}
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.2/dist/css/bootstrap.min.css" integrity="sha384-xOolHFLEh07PJGoPkLv1IbcEPTNtaed2xpHsD9ESMhqIYd0nLMwNLD69Npy4HI+N" crossorigin="anonymous">
+    <link rel="stylesheet" href="{% static 'css/styles.css' %}">
+  </head>
+  <body>
+    <div class="container-fluid">
+      <div class="row">
+        <div class="col-sm-2">
+          {% include 'includes/sidebar.html' %}
+        </div>
+        <div class="col-sm-10">
+          {% block welcome %}
+          {% endblock %}
+          {% block content %}
+          {% endblock %}
+        </div>
+      </div>
+    </div>
+  </body>
+</html>
```

Comment:
You need to add an empty line at the end of every HTML and CSS file

Reply:
idk, github was showing that red circle
![image](https://user-images.githubusercontent.com/123254785/226633443-e685ea94-6584-4d01-be55-1af4ae0efef3.png)

Reply:
Yup, I know, I even rechecked with reuploading.